src/chocolatey.resources/helpers/functions/Install-ChocolateyInstallPackage.ps1

```diff
@@ -258,13 +259,14 @@ param(
   $additionalInstallArgs = $env:chocolateyInstallArguments;
   if ($additionalInstallArgs -eq $null) {
     $additionalInstallArgs = '';
-  } else {
+  }
+  else {
     if ($additionalInstallArgs -match 'INSTALLDIR' -or `
-      $additionalInstallArgs -match 'TARGETDIR' -or `
-      $additionalInstallArgs -match 'dir\=' -or `
-      $additionalInstallArgs -match '\/D\='
+        $additionalInstallArgs -match 'TARGETDIR' -or `
+        $additionalInstallArgs -match 'dir\=' -or `
+        $additionalInstallArgs -match '\/D\='

```

Comment:
This could be done with a single -match:
```powershell
if ($additionalInstallArgs -match 'INSTALLDIR|TARGETDIR|dir\=|\/D\=') { ... }
```

Reply:
Can do. Should we go this route (I know what `|` means here), or should we leave it as is for readability?

Reply:
For readability we can opt for something like this instead:

```powershell
$argPattern = @(
    'INSTALLDIR'
    'TARGETDIR'
    'dir\='
    '\/D\='
) -join '|'

if ($additionalInstallArgs -match $argPattern) { ... }
```

That way we maintain a bit more readability here.

Reply:
To be clear... this PR is really about adding msp support, it isn't really about changing existing code, unless it is related to that change, so I don't think we need to spend too much time thinking about this change. We _can_ make changes to code outwith the change thing that is being worked on, but it should be done as a separate commit in the PR.  This is one of the things that I updated while rebasing this PR.